CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 cmake_minimum_required(VERSION 3.5)
-project(qt_gui_icons)
+project(tango_icons_vendor)
 
 find_package(ament_cmake REQUIRED)
 
-if (WIN32 OR APPLE OR INSTALL_QT_GUI_ICONS)
+if (WIN32 OR APPLE OR INSTALL_TANGO_ICONS)
```

Comment:
What is the purpose of this variable? If it should be used to configure the behavior of the package it should probably be an `option()`.

Reply:
The original point was just so people who wanted to install this package could override this if statement if they wanted to and install it manually. An option sounds good to me.